### ISSUE TITLE
Find Visible Edges

### ISSUE THREAD

**Issue by ArthurKo33** [LetsConstructIT/Grasshopper-TeklaDrawingLink#69]
Hi Greg.

Im trying my way in dimensioning Drawings. 
When I try dimension Plate that is not simple rectangle the component Find visible edges,
Find edges that it did not supposed to.

![image](https://github.com/user-attachments/assets/5494115e-8cbe-4077-9467-f9751d36ea3a)
![image](https://github.com/user-attachments/assets/fc9f52f3-028e-4d22-861e-63f6e5e2896e)
![image](https://github.com/user-attachments/assets/f53efbe4-244c-4c30-9313-d182f8a35761)

**Comment by olszewski-grzegorz** [comment]
@ArthurKo33 you're right, current implementation is 'lazy one'. The edges are taken into account individually, so they don't know about others. That's why when you look from the right side you are getting this one extra.

I will think about better algorithm, but not for the next release. It has to wait for 2025.
